src/sonic-yang-models/tests/yang_model_tests/tests_config/vrf.json

```diff
@@ -28,5 +28,16 @@
                 }]
             }
         }
+    },
```

Comment:
@tapashdas  Can you please add one more test for negative scenario (out of range)?

Reply:
done